src/client/components/MealReview/MealReviewDisplay.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import './mealreview.css';
+const EachMealReviewDisplay = ({ meal }) => {
```

Comment:
Over-explicit name. 
What is this component? The review of a meal? 
Then MealReview it is, right? 👌🏽

Reply:
Yes the review of a single meal to display.